package.json

```diff
@@ -32,14 +32,14 @@
     "@types/sinon": "^4.0.0",
     "chai": "^4.1.2",
     "del": "^3.0.0",
-    "gulp": "^3.9.1",
+    "gulp": "^4.0.2",
```

Comment:
This fails in node 11+ without a bump up to v4. Fortunately the `gulpclass` library had an update to support gulp v4 which makes this update seamless.